src/cellRenderers/SimpleCell.js

```diff
@@ -112,7 +111,11 @@ var SimpleCell = CellRenderer.extend('SimpleCell', {
             same = same && hoverColor === snapshot.colors[c++];
         }
 
-        // todo check if icons have changed
+        // VC-5714 instead of comparing the entire image object, we use id to check if the images has been updated.
+        same = same &&
+            leftIconId === snapshot.leftIconId &&
+            rightIconId === snapshot.rightIconId
```

Comment:
use icon id to check if there have any changes to the images, not sure if that is the best way.
open to discussion here.